hash-table/src/hash_table.c

```diff
@@ -129,8 +126,8 @@ static int ht_generic_hash(char* s, int a, int m) {
 
 
 static int ht_hash(char* s, int num_buckets, int attempt) {
-    int hash_a = ht_generic_hash(s, 151, num_buckets);
-    int hash_b = ht_generic_hash(s, 163, num_buckets);
+    int hash_a = ht_generic_hash(s, PRIME1, num_buckets);
```

Comment:
Magic numbers should be #defined symbolic constants at the top of the .c file (or maybe in the .h).